client/src/App.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PageRouters from './routers/PageRouters';
+import { QueryClient, QueryClientProvider } from 'react-query'
+import { ReactQueryDevtools } from 'react-query/devtools'
+import './App.scss';
+/* <!-- Css Styles --> */
+
+import "./assets/css/font-awesome.min.css"
```

Comment:
those imports should be placed in `./App.scss` then you use `import './App.scss'` is enough